psychopy/tests/test_data/test_utils.py

```diff
@@ -16,7 +16,7 @@ class Test_utilsClass:
     def test_importConditions(self):
         standard_files = []
         standard_files.append(join(fixturesPath, 'trialTypes.xlsx'))
-        standard_files.append(join(fixturesPath, 'trialTypes.xls'))
+        #standard_files.append(join(fixturesPath, 'trialTypes.xls')) # xls is depreciated
```

Comment:
I think this turned out not to be true - we misunderstood the pandas changelog to think that xls was no longer supported but actually the issue was that xlread is now only used for xls files (no longer for xlsx files) so xls is still fine